skiko/src/jsMain/kotlin/org/jetbrains/skia/impl/Native.js.kt

```diff
@@ -194,13 +194,27 @@ actual class InteropScope actual constructor() {
         return toInterop(interopPointers.toIntArray())
     }
 
+    actual fun toInterop(callback: (() -> Boolean)?): InteropPointer {
+        if (callback == null) { return 0 }
+
+        val data = CallbackData<Boolean>(null)
+        return _registerCallback({ data.value = callback() }, data)
```

Comment:
Where do we release callbacks?

Reply:
Dispose is performed on Cpp side, see `common_interop.h#L396`